tests/model_execution/test_bash.py

```diff
@@ -923,6 +937,18 @@ def setUpClass(cls):
             shutil.rmtree(cls.KPARSE_OUTPUT_FOLDER)
         os.makedirs(cls.KPARSE_OUTPUT_FOLDER)
 
+    # =============================================================================
+    # Custom GulCalc checks
+    # =============================================================================
+
+    def test_custom_gul_summarycalc_1_partition(self):
+        self.genbash("custom_gul_summarycalc_1_output", 1)
+        self.check("custom_gul_summarycalc_1_output_1_partition")
+    
+    def test_custom_gul_summarycalc_1_partition_chunk(self):
+        self.gen_chunked_bash("custom_gul_summarycalc_1_output", 1)
+        self.check_chunks("custom_gul_summarycalc_1_output_1_partition", 1)
+
```

Comment:
This will count the number of custom gulcalc processes that start and finish but won't actually make the script run them in the first place. To use a custom gulcalc executable, we need to modify the test class to pass through `get_getmodel_cmd` to `genbash()`.

I guess the checks pass because zero started == zero finished but if we're not using a custom gulcalc, there's no point checking whether it's crashed and that part could be disabled.

Alternatively if the check can be applied to the standard gulcalc as well, maybe the parameter names `custom_gulcalc_log_start` and `custom_gulcalc_log_finish` would be better as `gulcalc_log_start` and `gulcalc_log_finish`.